### PR TITLE
Generate Capistrano deploy target configs from chef sites data bag (#1049)

### PR DESCRIPTION
**Pull request by sethetter**
Relies on `knife data bag show sites` returning a list of site configurations.

### CODE REVIEW

File: bin/rb/cap_file_template.rb.erb [hunk 1]
```diff
@@ -0,0 +1,9 @@
+set :branch, 'dev' # Which branch you want to deploy from
```
Comment (doshitan) [inline bin/rb/cap_file_template.rb.erb]: Shouldn't this be configurable?

Comment (sethetter) [inline bin/rb/cap_file_template.rb.erb]: It certainly can be. I'll add a property to the site config for which branch to pull from and use that.

Comment (sethetter) [inline bin/rb/cap_file_template.rb.erb]: @doshitan d7ff437

**Comment by doshitan** [comment]
:+1:

**Pull request merged**